src/processors/processor.ts

```diff
@@ -46,8 +46,10 @@ export default class Processor {
     this.magicContent = new MagicString(content);
     this.styleParser = parser.bind(this);
 
+    if (!this.ast.css) throw new Error('ast.css is not defined.');
```

Comment:
Hack: Needed to do this to pass linting since the type of `Ast.css` changed to be nullable.